src/query/models/types.go

```diff
@@ -42,16 +42,6 @@ const (
 	// TypeDefault is an invalid scheme that indicates that the default scheme
 	// for the tag options version option should be used.
 	TypeDefault IDSchemeType = iota
-	// TypeLegacy describes a scheme where IDs are generated by appending
-	// tag name/value pairs with = and , separators. Note that an additional , is
-	// added to the end of the ID.
-	//
-	// NB: this should not be used, and exists here as a deprecated legacy
-	// ID generation scheme, as it may cause collisions in situations where
-	// incoming tags contain the following characters: << =," >>,  for example:
-	// {t1:v1},{t2:v2} -> t1=v1,t2=v2,
-	// {t1:v1,t2:v2}   -> t1=v1,t2=v2,
-	TypeLegacy
```

Comment:
Unfortunately we may need to keep this (maybe we can remove it and re-add it if people complain but not sure if that's the route we want to take)

Reply:
Decided to remove for now but will add back if necessary later.

Reply:
SG, ignore my comments on this then